### PR TITLE
Nag Hammadi Codex

### PR DESCRIPTION
**Pull request by Vostokapollo**
#### Summary
Content "Added the Nag Hammadi Codices to spiritual.json"

#### Purpose of change
It's the end of the world, nobody's gonna stop you from espousing heresy.

#### Describe the solution
I don't know how to code so I just copied the way the Talmud did variants.

#### Describe alternatives you've considered
Creating a new entry for every single codex, but that would have screwed up generation frequency and been annoying.

#### Testing
Worked fine in game. Text behaved as expected.

#### Additional context
Cantankerous Spuf on the Discord.

### CODE REVIEW

**Comment by ehughsbaird** [comment]
This cannot be merged until you add the `#### Summary` header on the line before the summary line.

File: data/json/items/book/spiritual.json [hunk 1]
```diff
@@ -117,6 +117,96 @@
...
+      {
+        "id": "holybook_nag_hammadi_03",
+        "name": { "str": "Nag Hammadi Codex III", "str_pl": "copies of the third Nag Hammadi Codex" },
+        "description": "It contains the Apocryphon of John (version 2), Holy Book of the Great Invisible Spirit, Eugnostos the Blessed, The Sophia of Jesus Christ, and the Dialogue of the Savior.",
+        "append": true
+      },
+      {
+        "id": "holybook_nag_hammadi_04",
+        "name": { "str": "Nag Hammadi Codex IV", "str_pl": "copies of the fourth Nag Hammadi Codex" },
+        "description": "It contains the Apocryphon of John (version 1) and Holy Book of the Great Invisible Spirit.",
```
Comment (Procyonae) [inline data/json/items/book/spiritual.json:151]: Is it intentional that both the texts you've mentioned in IV are also in II and III?

Comment (Vostokapollo) [reply data/json/items/book/spiritual.json:151]: Yes, the Nag Hammadi library has several copies of the same book, written by either different authors or copied by different individuals.

Comment (Procyonae) [reply data/json/items/book/spiritual.json:151]: That's cool I have no knowledge on the topic and were just checking you hadn't done it by mistake.
I don't see an issue with marking this as resolved as it's basically just noise. If the change leads to useful discussion say justifying the way you've done something over an alternative or just some useful stuff for future contributors I leave them unresolved so peeps can see them without digging but idk that there's any "etiquette".

**Pull request merged**